src/components/molecules/carousel/carousel.module.scss

```diff
@@ -9,7 +9,7 @@
 .carousel {
   overflow: hidden;
   height: 100%;
-  min-height: 10vh;
+  min-height: 350px;
```

Comment:
I don't think `min-height` or `min-width` should be a relative number, this is what caused the bad styling on small screens. It should be minimum 350px because any smaller than that will break the card styling.